appinventor/blocklyeditor/src/blocks/components.js

```diff
@@ -170,6 +170,13 @@ Blockly.Blocks.component_event = {
     if (this.instanceName == oldname) {
       this.instanceName = newname;
       this.componentDropDown.setValue(this.instanceName);
+      if (this.isCollapsed())
+      {
+        var COLLAPSED_INPUT_NAME = '_TEMP_COLLAPSED_INPUT';     
```

Comment:
@jisqyv, what variable name should be changed? 

```COLLAPSED_INPUT_NAME``` in this case is used as a constant string.

Is there a better way to write this code?

Thanks.